packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -61,7 +61,7 @@ class InteractiveViewer extends StatefulWidget {
   InteractiveViewer({
     super.key,
     this.clipBehavior = Clip.hardEdge,
-    this.alignPanAxis = false,
+    this.alignPanAxis = PanAxis.both,
```

Comment:
I wonder if it might be better to create a completely new parameter ("panAxis"?) so that we can use Flutter's normal deprecation flow.  That's probably the smoothest approach for anyone that might be using alignPanAxis right now.

Plus, I think panAxis is a better name than alignPanAxis now.  What do you think?

Here's an example of another deprecated parameter in another widget:

https://github.com/flutter/flutter/blob/6e95067273da1f886aa64cf475015f999f6ce817/packages/flutter/lib/src/material/app_bar.dart#L194-L198

Reply:
I thought about it but I waited until your review cause I wanted to hear the name of the new parameter by you. 

I think that's the best way to do it.

Reply:
@justinmc quick question, how do I know the version in which this parameter is being deprecated?

Reply:
Check the latest tag here: https://github.com/flutter/flutter/tags

Reply:
Found out today, now I'm using the correct tag 👌🏻